ihmc-pub-sub/build.gradle.kts

```diff
@@ -8,67 +11,65 @@ ihmc {
 
    configureDependencyResolution()
    javaDirectory("main", "../../swig/FastRTPS/generated")
-   javaDirectory("main", "../../build/generated/sources/xjc/java/main")
+   javaDirectory("xjc", "../xjc/java")
```

Comment:
I don't think you need this since you're using a source set.

Reply:
removed